.github/workflows/phpunit-mysql.yml

```diff
@@ -35,6 +35,11 @@ jobs:
         options: --health-cmd="mysqladmin ping" --health-interval 5s --health-timeout 2s --health-retries 5
 
     steps:
+      - name: Enable ONLY_FULL_GROUP_BY MySQL option
+        run: |
+          echo "SET GLOBAL sql_mode=(SELECT CONCAT(@@sql_mode,',ONLY_FULL_GROUP_BY'));" | mysql -h 127.0.0.1 -P 4444 -u root -prootpassword
```

Comment:
Can we not use https://mariadb.com/kb/en/server-system-variables/#sql_mode instead?
In the service start parameters?

Reply:
I use that in drone where I found out how to apply it:
https://github.com/nextcloud/spreed/pull/7032/files#diff-5b4c4090a99df0c5d2b12cf660513eb64bbd1b209c208c938bc6e0bef2400e7bR81

But I couldn't find the thing for github actions to pipe it down to the mysql running inside a docker.

Reply:
Pushed a patch, let's see